src/components/filterCheckbox/index.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { FC } from 'react';
+
+import { chakra } from '@chakra-ui/react';
+
+import Flex from '../flex';
+import Checkbox from '../checkbox';
+import Box from '../box';
+
+interface Item {
+  value: string | number;
+  label: string;
+}
+
+export interface Props {
+  name: string;
+  options: Item[];
+  selected?: (string | number)[];
+  onSelect?: (value: string | number) => void;
+  facet?: { [key: string]: number };
+  apply: (filtersToApply: unknown) => void;
+}
+
+const CheckboxFilter: FC<Props> = ({
+  name,
+  options,
+  selected = [],
+  onSelect,
+  facet,
+  apply: applyFilters,
+}) => {
+  const values = options.map(({ value }) => value);
+
+  const selectValue = (value: string | number) => () => {
+    const selectionCopy = [...selected];
+
+    if (onSelect) {
+      onSelect(value);
+    }
+
+    if (selectionCopy.includes(value)) {
+      const i = selectionCopy.indexOf(value);
+      selectionCopy.splice(i, 1);
+    } else {
+      selectionCopy.push(value);
+    }
+
+    selectionCopy.sort((a, b) =>
+      values.indexOf(a) < values.indexOf(b) ? -1 : 1
+    );
+
+    applyFilters({ [name]: selectionCopy });
+  };
+
+  const renderOption = ({ value, label }: Item) => {
+    const isSelected = selected.includes(value);
+    const resultsCount = facet?.[value.toString()] || 0;
+    const hasFacet = !!facet;
+    const isDisabled = hasFacet && resultsCount === 0 && !isSelected;
+
+    return (
+      <Box key={`filter_${name}_${value}`}>
+        <Checkbox
+          name={`filter_${name}_${value}`}
+          label={
+            <Flex width="5xl" justifyContent="space-between">
```

Comment:
correct me if I am wrong but shouldn't it just be 100%? We are going to have different sizes for the popover/model for desktop and mobile devices. I think we just want it to take the whole width